configs/recognition/i3d/i3d_nl_dot_product_r50_32x2x1_100e_kinetics400_rgb.py

```diff
@@ -5,7 +5,6 @@
 
 # model settings
 model = dict(
-    type='Recognizer3D',
```

Comment:
reason?

Reply:
it's inherited from _base_/models/i3d_r50.py. There is no need to specify the model type.